src/Yasumi/Provider/Ukraine.php

```diff
@@ -35,6 +38,13 @@ class Ukraine extends AbstractProvider
      */
     public const ID = 'UA';
 
+    /**
+     * Type definition for postponed holidays due to weekend holidays.
+     * Normally holidays on a weekend will be postponed to monday.
+     * These mondays will get this type.
+     */
+    public const TYPE_POSTPONED = 'postponed';
```

Comment:
Maybe this is a difference in language but holidays that fall in the weekend are often 'substituted' or observed on another day (for example the next Monday). In English and in the Yasumi library we use the term 'substitute'.
For this situation you would want to keep the current holiday (e.g. on Sunday) and add a new substitute holiday (e.g. Monday). You can use the SubstituteHoliday class for this.

Reply:
Thanks. Unfortunately, I didn't noticed the class `SubstituteHoliday` before.